test.md

```diff
@@ -19,14 +19,33 @@ module WASM-TEST
     imports WASM-TEXT
 ```
 
+Configuration
+-------------
+
+```k
+    configuration
+      <k> $PGM:Stmts </k>
+      <wasm/>
```

Comment:
We should always have a surrounding configuration for grabbing onto, so do `<wasm-test> <k> ... </k> <wasm/> </wasm-test>`

Reply:
Done